File: server/src/main/java/com/poly/service/DiscountService.java
package com.poly.service;

import com.poly.entity.Discount;
import java.util.List;

public interface DiscountService {

    List<Discount> getAllDiscount();

    Discount getDiscountByName(String name);

    Discount getDiscountById(Integer id);

    Discount create(Discount entity);

    Discount update(Discount entity, Integer id);
}

File: client/src/servers/discountService.js
import api from './axiosConfig';
import pathApi from '~/constants/pathApi';

const getAllDiscount = () => {
  return api.get(`${pathApi.discount}`);
};

const getDiscountById = (id) => {
  return api.get(`${pathApi.discount}/${id}`);
};


export { getAllDiscount, getDiscountById };
